hostingde/record_resource_test.go

```diff
@@ -52,6 +52,13 @@ resource "hostingde_record" "test_mx" {
   content = "mail.example2.test"
   priority = 10
 }
+resource "hostingde_record" "test_mx" {
```

Comment:
This looks like a duplicate of the above test resource. `make testacc` passes for me without this.

Reply:
You're right, removed it